components/forms/_checkboxes/checkbox.scss

```diff
@@ -106,3 +106,14 @@
     top: 6px;
   }
 }
+.msds-fieldset {
```

Comment:
I would move all of this to the main solution general.scss

and we work with px instead of rem. After npm compiles all px are converted to rem :)